cishouseholds/pipeline/covid_event_transformations.py

```diff
@@ -782,4 +783,11 @@ def derive_contact_any_covid_covid_variables(df: DataFrame) -> DataFrame:
         "visit_datetime",
     )
 
+    df = assign_grouped_variable_from_days_since_contact(
+        df=df,
+        reference_column="contact_known_or_suspected",
```

Comment:
column name is contact_known_or_suspected_covid (derived on line 753 of covid_event_transformations)